bash_it.sh

```diff
@@ -1,37 +1,48 @@
 #!/usr/bin/env bash
 # Initialize Bash It
+BASH_IT_LOG_PREFIX="core: main: "
 
 # Only set $BASH_IT if it's not already set
 if [ -z "$BASH_IT" ];
 then
   # Setting $BASH to maintain backwards compatibility
-  # TODO: warn users that they should upgrade their .bash_profile
   export BASH_IT=$BASH
   BASH="$(bash -c 'echo $BASH')"
   export BASH
+  BASH_IT_OLD_BASH_SETUP=true
 fi
 
+# Load composure first, so we support function metadata
+# shellcheck source=./lib/composure.bash
+source "${BASH_IT}/lib/composure.bash"
+# We need to load logging module first as well in order to be able to log
+source "${BASH_IT}/lib/log.bash"
```

Comment:
Can you please add a `shellcheck source=...` comment above this line, so that `shellcheck` can find the referenced file? See line 16 for an example...

Reply:
done